Key Error after upgrade to 2.1.0

After the upgrade to 2.1.0 I observe the following error:


```
    def _parse_xml(self, xml_filename):
        root = ElementTree.parse(xml_filename)
        result = []
        for test_suite in root.findall("testsuite"):
            test_suite_name = test_suite.attrib["name"]
            for test_case in test_suite.findall("testcase"):
                test_name = test_case.attrib["name"]
                failures = []
                failure_elements = test_case.findall("failure")
                for failure_elem in failure_elements:
                    failures.append(failure_elem.text)
                skippeds = []
>               if test_case.attrib["result"] == "skipped":
E               KeyError: 'result'

test_env\lib\site-packages\pytest_cpp\google.py:125: KeyError
```

It looks like replacing  test_case.attrib.get("result", None) with  test_case.attrib["result"]  is the issue: https://github.com/pytest-dev/pytest-cpp/commit/2a2f12568f0611c6263bf8de9420d59d6d7f4b32

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+# 2.1.1 (2021-12-06)
+
+- Fix corner-case issue when no result attribute is found in google test XML output file.
+
 # 2.1.0 (2021-12-03)
 
 - Added support for `GTEST_SKIP()` skipped message from Google Test `1.11`.

--- a/src/pytest_cpp/google.py
+++ b/src/pytest_cpp/google.py
@@ -122,7 +122,7 @@ class GoogleTestFacade(object):
                 for failure_elem in failure_elements:
                     failures.append(failure_elem.text)
                 skippeds = []
-                if test_case.attrib["result"] == "skipped":
+                if test_case.attrib.get("result", None) == "skipped":
                     # In gtest 1.11 a skipped message was added to
                     # the output file
                     skipped_elements = test_case.findall("skipped")
@@ -133,7 +133,7 @@ class GoogleTestFacade(object):
                     # append a "skipped" keyword
                     if not skipped_elements:
                         skippeds.append("Skipped")
-                if test_case.attrib["status"] == "notrun":
+                elif test_case.attrib.get("status", None) == "notrun":
                     skippeds.append("Disabled")
                 result.append((test_suite_name + "." + test_name, failures, skippeds))
 

